### PR TITLE
Syncing bugfixes

### PR DESCRIPTION
**Pull request by MatusKysel**

### CODE REVIEW

File: src/network/taraxa_capability.cpp [hunk 1]
```diff
@@ -539,10 +540,6 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
       auto it = _r.begin();
       bool is_final_sync_packet = (*it++).toInt<unsigned>();
 
-      if (is_final_sync_packet) {
-        syncing_state_.set_dag_syncing(false);
-      }
```
Comment (JakubFornadel) [inline src/network/taraxa_capability.cpp:544]: why is this deleted ?

Comment (MatusKysel) [reply src/network/taraxa_capability.cpp:544]: because is there again https://github.com/Taraxa-project/taraxa-node/pull/998/files#diff-7ee56eeb9607542d6ee2cdeb2a2c87697b6a45d964931b90e0ffe99c23461926R570 I think it was an issue of some merging

**Review by JakubFornadel — APPROVED** [review]

**Review by JakubFornadel — APPROVED** [review]

**Review by kstdl — APPROVED** [review]
